library/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,8 @@
         </intent>
     </queries>
 
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
```

Comment:
Shouldn't it have android:maxSdkVersion set? PR description suggests that it is only needed for API < 26.

Reply:
TIL - didn't know about this property.